packages/react/src/primitives/Link/Link.tsx

```diff
@@ -25,6 +24,7 @@ const LinkPrimitive: Primitive<LinkProps, 'a'> = (
       {...rest}
     >
       {children}
+      {isExternal && <IconBoxArrowUpRight />}
```

Comment:
Few things here:
- Prefer ternary returning `null` if `isExternal` is falsy to prevent this expression from evaluating to `false`
- How do i hide the icon?
- How do i position the icon to the left of the `children`?

Reply:
1. Updated to use a ternary operator
2. You can hide the icon when `isExternal` prop is set to `false` (default)
3. It's important to start with a word that indicates the action that results if customers click the link. Putting an icon at the beginning of the link disrupts the readability of the link quite massively, therefore it is a common convention to put icons after the actual link text

Reply:
> You can hide the icon when isExternal prop is set to false (default)

Sure, but what if the link is external and i don't want the icon?

> It's important to start with a word that indicates the action that results if customers click the link. Putting an icon at the beginning of the link disrupts the readability of the link quite massively, therefore it is a common convention to put icons after the actual link text

1. What abt RTL languages?
2. This response is an opinion on the usage of the `Link` component but does not address the actual question

Reply:
Good call on RTL languages. I'll work on adding a position prop, so devs can control the icon position

Reply:
There is no need to create a position prop since the direction='rtl' can be set in the `ThemeProvider` and it will automatically applied to the `Link` component
![Screenshot 2023-12-04 at 4 00 24 PM](https://github.com/aws-amplify/amplify-ui/assets/19784835/0d9a7565-8994-4ebb-96e1-6a57c6aa7d24)

Reply:
@lavr001 The above question is still unanswered:

> > You can hide the icon when isExternal prop is set to false (default)
> 
> Sure, but what if the link is external and i don't want the icon?

Reply:
@calebpollman Are you suggesting having a new prop called `hasIcon` that would be visible only when the link is external, so users can control if they want the icon or not?

Reply:
@calebpollman Added new prop `hideIcon`, so now if the Link is external you have an option to show/hide the icon. Also added a new prop `linkIconPosition`, so you can position the icon left or right to the Link
<img width="453" alt="Screenshot 2024-01-09 at 2 54 22 PM" src="https://github.com/aws-amplify/amplify-ui/assets/19784835/f533d422-d5f6-484a-b1b1-57388189d80b">